docs/changelog.md

```diff
@@ -2,7 +2,16 @@
 
 ## CARET
 
-### v0.4.0 <small>\_ Dec 16, 2022</small> {id = "0.3.4"}
+### v0.4.1 <small>\_ Dec 26, 2022</small> {id = "0.4.1"}
+
+- **New**: Added functions to assign intra-node data path with `message_context` ([CARET_analyze #196](https://github.com/tier4/CARET_analyze/pull/196))
+- **Update**: Changed design of visualization API for refactoring to avoid usage of Pandas ([CARET_analyze #116](https://github.com/tier4/CARET_analyze/pull/116), [CARET_analyze #167](https://github.com/tier4/CARET_analyze/pull/167), [CARET_analyze #168](https://github.com/tier4/CARET_analyze/pull/168), [CARET_analyze #170](https://github.com/tier4/CARET_analyze/pull/170))
+- Added option to control log level to suppress warning message ([CARET_analyze #220](https://github.com/tier4/CARET_analyze/pull/220))
```

Comment:
This PR adds a configure file rather than options.

```suggestion
- Added configure file to control log level to suppress warning message ([CARET_analyze #220](https://github.com/tier4/CARET_analyze/pull/220))
```